bundles/framework/myplacesimport/UserLayersList.jsx

```diff
@@ -46,6 +47,12 @@ export const UserLayersList = ({ data = [], controller, loading }) => {
             dataIndex: 'description',
             sorter: getSorterFor('description')
         },
+        {
+            align: 'left',
+            title: <Message messageKey='tab.grid.createDate' />,
+            dataIndex: 'created',
+            sorter: getSorterFor('created')
```

Comment:
We should use the render function for dates on the table instead of modifying the incoming list. If we ever do paging this reduces the amount of mapping we need to do and should make sorting more predictable (though `getSorterFor()` might not work with dates so we might need to change that).
```suggestion
            sorter: getSorterFor('created'),
            render: text => Oskari.util.formatDate(text)
```